src/props.ts

```diff
@@ -16,13 +15,17 @@ const INITIAL_PROPS = {
 }
 
 export const props$ = (options: WidgetOptions) => {
-  const oSites = options.sites || []
-  const sites$ = from(oSites.map(site => site.siteId)).pipe(
-    mergeMap(fetchSite),
-    toArray()
-  )
+  const sites = (options.sites || []).map(site => ({
+    id: site.siteId,
+    name: site.name,
+    title: site.title,
+    buildHookId: site.buildHookId,
+    url: site.name && `${site.hostname}`,
+    adminUrl: site.name && `${site.hostname}`,
```

Comment:
Shouldn't we remove `adminUrl` now?

Reply:
Isn't it good to have it there? Since we can derive it from the siteId?

Reply:
If there was a need before, nothing is stopping us from still having it now too.

Reply:
> Seems like it's not quite done?

There were some issues yes, should be sorted out now.